install/gce-installation-guide.md

```diff
@@ -0,0 +1,119 @@
+# Installing Kata Containers on Google Compute Engine
+
+Kata Containers on Google Compute Engine (GCE) makes used of [nested virtualization](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances). Most of the installation procedure is identical to that for Kata on your preferred distribution, but enabling nested virtualization currently requires extra steps on GCE. This guide walks you through creating an image and instance with nested virtualization enabled.
+
+As a pre-requisite this guide assumes you have installed and configured [Google Cloud SDK](https://cloud.google.com/sdk/downloads). For a zero-configuration option, all of the commands below have been tested under [Google Cloud Shell](https://cloud.google.com/shell/).
```

Comment:
If we want this doc to be a "category (2)" (see below), could we add a test like the following here?:

```
    ```bash
    $ command -v gcloud || { echo "ERROR: no Google Cloud Shell"; exit 1; }
    ```
```

Reply:
I can do that here, but I'm not sure we want this to be category (2) -- many of the commands below will prompt the user during their execution. That said, I've added some prose suggesting they verify their gcloud installation and config.